Competition2017/src/competition/subsystems/autonomous/DriveToBoilerUsingHeuristicsWithVisionCommandGroup.java

```diff
@@ -15,7 +17,10 @@ public DriveToBoilerUsingHeuristicsWithVisionCommandGroup(
             XPropertyManager propMan,
             RotateRobotToBoilerCommand rotateToBoilerCommand,
             CalculateDirectBoilerDriveDeltaCommand calculateCommand,
-            DriveToPointUsingHeuristicsCommand driveCommand){
+            DriveToPointUsingHeuristicsCommand driveCommand,
+            ShiftGearCommand shiftCommand){
+        shiftCommand.setGear(Gear.LOW_GEAR);
+        addSequential(shiftCommand);
```

Comment:
I don't think shiftGearCommand will currently end so this would hold everything else up, maybe add a short timeout, 100ms?